scripts/example.trc

```diff
@@ -1,16 +1,16 @@
-core   0: 0x80001048 (0x00007179) c.addi16sp sp, -48
-core   0: 0x8000104a (0x0000d622) c.swsp  s0, 44(sp)
-core   0: 0x8000104c (0x00001800) c.addi4spn s0, sp, 48
-core   0: 0x8000104e (0xfca42e23) sw      a0, -36(s0)
-core   0: 0x80001052 (0xfcb42c23) sw      a1, -40(s0)
-core   0: 0x80001056 (0x000047cd) c.li    a5, 19
-core   0: 0x80001058 (0xfef42623) sw      a5, -20(s0)
-core   0: 0x8000105c (0xfec42783) lw      a5, -20(s0)
-core   0: 0x80001060 (0x000007d5) c.addi  a5, 21
-core   0: 0x80001062 (0xd007f7d3) fcvt.s.w fa5, a5
-core   0: 0x80001066 (0xfef42427) fsw     fa5, -24(s0)
-core   0: 0x8000106a (0x00004781) c.li    a5, 0
-core   0: 0x8000106c (0x0000853e) c.mv    a0, a5
-core   0: 0x8000106e (0x00005432) c.lwsp  s0, 44(sp)
-core   0: 0x80001070 (0x00006145) c.addi16sp sp, 48
-core   0: 0x80001072 (0x00008082) ret
+0x80001048 c.addi16sp
```

Comment:
Has the trace format really changed? There are no corresponding changes to the Makefile in this PR.

Reply:
The Makefile changes in the _MakeFile-Rework_ PR haven't been merged so I wasn't able to build from them in this PR; instead I've just pushed the changes to the _MakeFile-Rework_ PR (which probably wasn't the greatest of ideas). What would've been a better way for me to make the Makefile changes considering that in this PR, the Makefile is completely unchanged and so it would not have any functionality?